### DISCUSSION TITLE
求助：参考文献编号右对齐，编号与作者之间的距离调整

### DISCUSSION THREAD

**Discussion by puppet9527** [zotero-chinese/wiki#231] [求助 / Q&A] [answered]
![Clip_2024-07-08_20-36-10](https://github.com/zotero-chinese/wiki/assets/42835379/54a06eb9-915e-4cdd-a19d-63334a2bdf4b)
使用的是这个样式：gb-t-7714-2015-numeric-bilingual-no-url-doi

**Comment by northword** [comment, marked as answer]
切换一次样式，从这个切换到另一任意个，再切换回来

**Reply by zepinglee** [reply]
不是应该增加段落悬挂缩进和制表符宽度的距离吗？

**Reply by northword** [reply]
从99到100这种进一位的没有对齐，通过切换一次样式，zotero会重新计算制表符的宽度，就正常了，按理说应该在添加引注的时候就要重新计算了，属于老bug了。

当然手动调整悬挂和制表符宽度也可以，但不少同学其实搞不明白制表符怎么搞，反而切换一次显得更方便。

**Reply by northword** [reply]
之前zo推过一个提交修复这个问题，但是会导致每次添加引注会覆盖的用户设置的行距等格式，就给revert了

https://forums.zotero.org/discussion/106215/zotero-7-refreshing-the-bibliography-will-overwrite-the-bibliography-style-in-word#latest

**Reply by zepinglee** [reply]
原来会重新计算制表符……我一直都是手动在 Word “样式窗格”中修改“书目”的“制表位”。

**Reply by Alphasxd** [reply]
为啥切换了其他样式，还是不能解决啊，不止[99]到[100]，从[9]到[10]就已经出现这个问题了
